docset/winserver2025-ps/smbshare/Set-SmbServerConfiguration.md

```diff
@@ -183,9 +185,10 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -AuditSmb1Access
+### -AuditClientDoesNotSupportEncryption
```

Comment:
Enables auditing of SMB clients that do not support encryption. Clients that connect and do not list SMB encryption in their supported capabilities will be recorded in the Windows event log.